### PR TITLE
Use Logger class and static methods

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
Parent issue: #662 
Previously, we tried to use singleton Logger. (#663)
However, code needs `Logger.getInstance().outputWithTime(msg)`, which seems relatively long.

Instead let's use `Logger.outputWithTime(msg)`.

This commit introduces three changes.

1. Make methods in `Logger` as `static` to shorten its usage: e.g., `Logger.outputWithTime(msg)`
2. From 1, we don't have any `logger` object. 
    - Do not pass `logger` as param any more.
    - Do not keep `logger` as object variable
3. Change `console.log()` to `Logger.outputWithTime()`.

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

File: src/Utils/Logger.ts [hunk 1]
```diff
@@ -34,27 +36,27 @@ export class Logger {
...
-    this.checkShow();
-    this.outputChannel.appendLine('[' + dateTime.toLocaleString() + '] ' + msg);
+    Logger.checkShow();
+    Logger.outputChannel.appendLine('[' + dateTime.toLocaleString() + '] ' + msg);
   }
 
-  public output(msg: string) {
-    this.checkShow();
-    this.outputChannel.append(msg);
+  public static output(msg: string) {
```
Comment (dayo09) [inline src/Utils/Logger.ts:53]: (Q) What about `Logger.log()` or `Logger.logLine` `Logger.logWithTime`? For (1) It sounds better with verb at the front and (2) `log` sounds familiar than `output` (ex. console."log"). BTW I understand that Logger logs just like console.log but into its

**Review by jyoungyun — APPROVED** [review]
LGTM

(suggestion)
How about adding `TAG` about logs?
Or it is another way to create a separate logger for each TAG.

**Comment by hyunsik-yoon** [comment]
@dayo09 @jyoungyun Thanks for good opinion. I think I have to do some research about logger. 😸

**Comment by YongseopKim** [comment]
> How about adding TAG about logs?
Or it is another way to create a separate logger for each TAG.

It's very good idea. Android does like [this](https://developer.android.com/studio/debug/am-logcat?hl=ko#WriteLogs).

**Comment by hyunsik-yoon** [comment]
Let me close this. Please refer to #700.

**Pull request closed without merging**